### ISSUE TITLE
[Feature-request] Set task estimation

### ISSUE THREAD

**Issue by dzhokn** [ti-dev/Scrum-it#3]
Currently, each task is estimated on 8hrs by default. I would love "ScrumIt" if it provided SetTaskEstimation feature. It's easy to be implemented. If I have time to set suitable environment I might do it myself.

**Comment by ti-dev** [comment]
Yes, it is set to 8 (hours / points, however you wanna call it). I already thought about this feature and it would be great if there is a UI possibility to flip the task on the scrum board and then on the other side of the task, there are some possibilities to edit the task like changing the hours.
